packages/admin/cms-admin/src/dam/DataGrid/thumbnail/DamThumbnail.tsx

```diff
@@ -37,10 +47,38 @@ interface DamThumbnailProps {
 }
 
 export const DamThumbnail = ({ asset }: DamThumbnailProps): React.ReactElement => {
+    const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+    const open = Boolean(anchorEl);
+
+    const handleMouseOver = (event: React.MouseEvent<HTMLElement>) => {
+        setAnchorEl(event.currentTarget);
+    };
+    const handleMouseLeave = () => {
+        setAnchorEl(null);
+    };
+
     let thumbnail;
     if (asset.__typename === "DamFile") {
         if (asset.mimetype.startsWith("image/") && asset.image && asset.image.thumbnailUrl) {
-            thumbnail = <ImageThumbnail src={asset.image.thumbnailUrl} />;
+            const id = open ? asset.fileUrl : undefined;
+            thumbnail = (
+                <>
+                    <ImageThumbnail
+                        aria-describedby={id}
```

Comment:
We don't provide ARIA hints anywhere in the admin. Should we really add them here? Or is this a leftover copied from the MUI docs?

Reply:
Also I think `aria-describedby` expects the ID of an element. I don't think putting a URL in there does anything.

Reply:
It's a leftover. I removed it 👍🏻